contracts/base/Transfers.sol

```diff
@@ -28,7 +30,7 @@ abstract contract Transfers is ITransfers {
         }
     }
 
-    function refundETH() external payable override {
+    function refundETH() public payable override {
```

Comment:
Why did you make this public? It was external before so that it can be called from outside of the smart contract. Public allows it to be called from outside the smart contract (like external) but also from inside the smart contract without requiring a reference to "this" instance. For that reason public consumes more memory. Was using external before to lower the memory footprint since memory costs gas and gas can get pretty expensive.

Reply:
Making it external is affecting the call of the function in TestTransafers.sol. Getting following error.

DeclarationError: Undeclared identifier.
  --> contracts/test/TestTransfers.sol:20:9:
   |
20 |         refundETH();
   |         ^^^^^^^^^

Reply:
right, that's because you don't need testRefundETH() to test refundETH(). Just get rid of that function.